Makefile

```diff
@@ -286,7 +286,7 @@ release-manager-release: ## - Builds a snapshot release. The Go version defined
 ## get-version : Get the Fleet server version
 .PHONY: get-version
 get-version:
-	@echo $(DEFAULT_VERSION)
+	@echo $(VERSION)
```

Comment:
`$VERSION` differs from `$DEFAULT_VERSION` as it may include snapshot or other version indicators.
The only other target (besides cloude2e) that uses `get-version` is the buildkite dra target, which should filter out the snapshot indicator